spec/features/finding/index_details_spec.rb

```diff
@@ -2,24 +2,33 @@
 
 RSpec.feature "Index details" do
   scenario do
-    given_there_is_an_edition
+    given_there_is_an_edition_in_my_organisation
     when_i_visit_the_index_page
-    then_i_can_see_the_title
+    then_i_am_redirected_to_my_organisation_filter
+    and_i_can_see_the_title
     and_i_can_see_the_document_type
     and_i_can_see_the_state
     and_i_can_see_who_last_edited_it
   end
 
-  def given_there_is_an_edition
+  def given_there_is_an_edition_in_my_organisation
     @editor = create(:user)
-    @edition = create(:edition, last_edited_by: @editor)
+    @edition = create(:edition,
+                      last_edited_by: @editor,
+                      tags: {
+                        primary_publishing_organisation: [User.first.organisation_content_id],
+                      })
   end
 
   def when_i_visit_the_index_page
     visit documents_path
   end
 
-  def then_i_can_see_the_title
+  def then_i_am_redirected_to_my_organisation_filter
+    expect(page).to have_current_path(documents_path(organisation: User.first.organisation_content_id))
```

Comment:
Would this help?

https://github.com/teamcapybara/capybara/blob/master/lib/capybara/spec/session/has_current_path_spec.rb#L80

Reply:
Yes - great find 👍